### PR TITLE
MON-19264 - Documentation - Depreciate the ip label datametrie monitoring connector

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Mark the ip label datametrie monitoring connector as depreciate

## Target version (i.e. version that this PR changes)

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [ ] 23.04.x (staging)
- [ ] Cloud (staging)
- [x] Monitoring Connectors (staging)
- [ ] 23.10.x (next)

### CODE REVIEW

**Comment by lucie-dubrunfaut** [comment]
New PR link to https://github.com/centreon/centreon-documentation/pull/2416

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-iplabel-datametrie-restapi.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ title: IP-Label datametrie API
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+## Attention
+
+Ce plugin-pack a été référencé obsolète, désormais le service Datamétrie
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-iplabel-datametrie-restapi.md:10]:
```suggestion
Ce connecteur est obsolète, car le service Datamétrie 
```

File: pp/integrations/plugin-packs/procedures/applications-monitoring-iplabel-datametrie-restapi.md [hunk 2]
```diff
@@ -5,6 +5,10 @@ title: IP-Label datametrie API
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+## Warning
+
+This plugin pack has been marked as deprecated, since the Datamétrie online
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-monitoring-iplabel-datametrie-restapi.md:10]:
```suggestion
This connector is deprecated, as the Datamétrie online 
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-iplabel-datametrie-restapi.md [hunk 3]
```diff
@@ -5,6 +5,10 @@ title: IP-Label datametrie API
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+## Attention
+
+Ce connecteur est obsolète, car le service Datamétrie 
+online n'est plus disponible.
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-iplabel-datametrie-restapi.md:11]:
```suggestion
> Ce connecteur est obsolète, car le service Datamétrie 
> online n'est plus disponible.
```

File: pp/integrations/plugin-packs/procedures/applications-monitoring-iplabel-datametrie-restapi.md [hunk 4]
```diff
@@ -5,6 +5,10 @@ title: IP-Label datametrie API
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+## Warning
+
+This connector is deprecated, as the Datamétrie online 
+service is not available any more.
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/applications-monitoring-iplabel-datametrie-restapi.md:11]:
```suggestion
> This connector is deprecated, as the Datamétrie online 
> service is not available any more.
```

**Review by omercier — CHANGES_REQUESTED** [review]

**Review by cg-tw — DISMISSED** [review]

**Review by omercier — APPROVED** [review]

**Comment by lucie-dubrunfaut** [comment]
Can't merge because of "continuous-integration/jenkins/pr-merge"
Can't fix it on local command line instructions :
`remote: error: GH006: Protected branch update failed for refs/heads/staging.
remote: error: Changes must be made through a pull request. Required status check "continuous-integration/jenkins/pr-merge" is errored.
To github.com:centreon/centreon-documentation.git
 ! [remote rejected]     staging -> staging (protected branch hook declined)
error: failed to push some refs to 'github.com:centreon/centreon-documentation.git'
`

**Review by cg-tw — APPROVED** [review]

**Review by omercier — APPROVED** [review]

**Pull request merged**